static/js/translation.js

```diff
@@ -0,0 +1,221 @@
+var BATCH_SIZE = 10;
+var batches = [];
+var batch_queue = [];
+var sent_map = {};
+var pg_map = {};
+var is_translating = false;
+
+function hoverInTranslation(ev) {
+  var sId = $(this).parent().attr("id");
+  if (sId === null || sId === undefined || !sent_map || !sId.startsWith("trnsl")) {
+    // No id: nothing to do
+    return;
+  }
+  var offset = $(this).position();
+  height = $(this).outerHeight();
+  bottom = offset.top + height;
+  sent_idx = sId.replace("trnsl", "");
+  $("#info-trnsl").html(sent_map[sent_idx]["is"]);
+  // Position the info popup
+  $("#info-trnsl")
+    .css("top", bottom + "px")
+    .css("left", offset.left.toString() + "px")
+    .css("visibility", "visible");
+}
+
+function hoverOutTranslation() {
+  $("#info-trnsl").css("visibility", "hidden");
```

Comment:
Can be merged into one line: `$("#info-trnsl").css("visibility", "hidden").html("");`